react/src/components/organisms/TabContainer/index.js

```diff
@@ -34,16 +63,36 @@ class TabContainer extends React.Component {
     const active = defaultTab;
     this.childrenWithProps = React.Children.map(children, (child, index) => {
```

Comment:
The index here can be used to set the correct tabindex per tab. If you make a new property below on the cloneElement line, you could pass the index and then pull it out of tab's props.